.github/workflows/Build.yml

```diff
@@ -313,8 +313,6 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - name: ⌛ Upload to 🐋 quay.io
         run: |
-          pip install setuptools
-          version=$(python3 setup.py --version)
```

Comment:
Why did u delete the version ?

Reply:
we always use the latest version, so there is no need for it.

Reply:
We are publishing to the public quay so build and push must be there.
I don't find extra spaces.